tests/dummy/app/templates/index.hbs

```diff
@@ -83,6 +83,16 @@
         be included in your build
       </td>
     </tr>
+    <tr>
+      <td><code>notification</code></td>
+      <td><code>Object</code></td>
+      <td><code>{}</code></td>
+      <td>
+        Global options for the {{#link-to 'services.notification'}}notification service{{/link-to}}.
+        For available options (<code>message</code> and <code>status</code> will be ignored)
+        check the <a href="https://getuikit.com/docs/notification#component-options">UIkit documentation</a>.
```

Comment:
For available options see the UIkit documentation. `message` and `status` will be ignored, because they are controlled by the addon.